Dockerfile

```diff
@@ -1,98 +1,23 @@
-    rm mconda-install.sh && \
-    /miniconda/bin/conda create -y -q -n sqlflow-dev python=3.6 ${CONDA_ADD_PACKAGES} && \
-    echo ". /miniconda/etc/profile.d/conda.sh" >> ~/.bashrc && \
-    echo "source activate sqlflow-dev" >> ~/.bashrc && \
-    /bin/bash -c "source /miniconda/bin/activate sqlflow-dev && python -m pip install \
-    numpy==1.16.1 \
-    tensorflow==2.0.0-alpha0 \
-    mysql-connector-python \
-    impyla \
-    pyodps \
-    jupyter \
-    sqlflow \
-    pre-commit \
-    ${PIP_ADD_PACKAGES} \
-    "
-ENV PATH="/miniconda/bin:$PATH"
-
-# Install go 1.11.5
-RUN apt-get install -y wget git && \
-    wget --quiet https://dl.google.com/go/go1.11.5.linux-amd64.tar.gz && \
-    tar -C /usr/local -xzf go1.11.5.linux-amd64.tar.gz && \
-    rm go1.11.5.linux-amd64.tar.gz && \
-    mkdir -p /go
 ENV GOPATH /go
-ENV PATH $PATH:/usr/local/go/bin:/go/bin
-
-RUN go get github.com/golang/protobuf/protoc-gen-go && \
-    mv /go/bin/protoc-gen-go /usr/local/bin/ && \
-    go get golang.org/x/lint/golint
-
-# Install protobuf compiler
-RUN wget --quiet https://github.com/protocolbuffers/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip && \
-    unzip -qq protoc-3.6.1-linux-x86_64.zip -d /usr/local && \
-    rm protoc-3.6.1-linux-x86_64.zip
-
-# install mysql without a password prompt
-RUN echo 'mysql-server mysql-server/root_password password root' | debconf-set-selections && \
-    echo 'mysql-server mysql-server/root_password_again password root' | debconf-set-selections && \
-    apt-get install -y mysql-server && \
-    mkdir -p /var/run/mysqld && \
-    mkdir -p /var/lib/mysql && \
-    chown mysql:mysql /var/run/mysqld && \
-    mkdir -p /docker-entrypoint-initdb.d
-VOLUME /var/lib/mysql
-
-# Add the python module of sqlflow to PYTHONPATH
-ENV PYTHONPATH $PYTHONPATH:$GOPATH/src/github.com/sql-machine-learning/sqlflow/sql/python
-
-# Build SQLFlow binaries by git clone the latest develop branch.
-# During development, /go will be overridden by -v.
-RUN mkdir -p /go/src/github.com/sql-machine-learning && \
-    cd /go/src/github.com/sql-machine-learning && \
-    git clone -q https://github.com/sql-machine-learning/sqlflow.git && \
-    cd sqlflow && \
-    go generate ./... && \
-    go get -v -t ./... && \
-    go install -v ./... && \
-    cd /
-
-# Install latest sqlflow_models for testing custom models, see main_test.go:CaseTrainCustomModel
-RUN git clone https://github.com/sql-machine-learning/models.git && \
-    cd models && \
-    bash -c "source activate sqlflow-dev && python setup.py install" && \
-    cd ..
-
-# Fix jupyter server "connecting to kernel" problem
-# https://github.com/jupyter/notebook/issues/2664#issuecomment-468954423
-RUN /bin/bash -c "source activate sqlflow-dev && python -m pip install tornado==4.5.3"
-
-# Load sqlflow Jupyter magic command automatically. c.f. https://stackoverflow.com/a/32683001.
+ENV PATH /miniconda/bin:/miniconda/envs/sqlflow-dev/bin:/usr/local/go/bin:/go/bin:$PATH
```

Comment:
`/miniconda/bin:/miniconda/envs/sqlflow-dev/bin` => `/miniconda/envs/sqlflow-dev/bin:/miniconda/bin`.

Otherwise, `sqlflow-dev` virtual environment will be hidden..

Reply:
Thanks, done.